less-builder.js

```diff
@@ -20,33 +36,63 @@ var cssInject = [
 ].join(';');
 
 exports.bundle = function (loads, compileOpts, outputOpts) {
-	var _this = this;
+    var loader = this;
+    var writeStubs = typeof outputOpts == 'undefined';
+    outputOpts = outputOpts || compileOpts;
+
+    var stubDefines = writeStubs ? loads.map(function(load) {
+      return (compileOpts.systemGlobal || 'System') + ".register('" + load.name + "', [], false, function() {});";
```

Comment:
I know, but you were missing the concatenation operator.
https://github.com/HuasoFoundries/systemjs-less-plugin/pull/2/commits/cb846f9f1aa19d95cacf142437a7ae6300718dd7#diff-eee973e88399de0b3160a500ff489e29R26